### PR TITLE
cpexId Id Module: Renamed to czechAdId [vendor confirms no use yet, non-breaking]

### PR DESCRIPTION
**Pull request by EskelCz**
## Type of change
- [X] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [X] Other

## Description of change
Simply renaming of the module. The new website (linked in readme) is about to go live in a few days.

Documentation PR: https://github.com/prebid/prebid.github.io/pull/4286

### CODE REVIEW

**Comment by patmmccann** [comment]
appears to be breaking, tagging as #8539

**Comment by EskelCz** [comment]
@patmmccann Sorry I don't understand, it's not passing tests of some future version? What should I do?

**Comment by bretg** [comment]
@EskelCz  - the issue is that Prebid does not allow renaming of modules except when there's a major release. That would cause problems for publishers using the old name. We guarantee to pubs that they don't have to change their in-page or build procedures except at major releases.

So in your case, one approach would be be add an "alias" for your user ID module so that both the old name and the new name work. @dgirardi can help you determine how best to do that.

File: modules/czechAdIdSystem.js [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
-export const storage = getStorageManager({ gvlid: 570, moduleName: 'cpexId' })
+export const storage = getStorageManager({ gvlid: 570, moduleName: 'czechAdId' })
 
 // Returns the id string from either cookie or localstorage
 const readId = () => { return storage.getCookie('czaid') || storage.getDataFromLocalStorage('czaid') }
 
 /** @type {Submodule} */
-export const cpexIdSubmodule = {
-  version: '0.0.5',
+export const czechAdIdSubmodule = {
```
Comment (dgirardi) [inline modules/czechAdIdSystem.js:18]: my strategy would be to export this as a function taking in the module name, and have a "dummy" `cpexIdSystyem.js` that re-registers under the old name until version 8. alternatively, you could delay this change until v8.

**Comment by EskelCz** [comment]
@bretg I see. The thing is, our ID module is in testing phase and not yet fully public. In order for publishers to use it, they must be connected to the Czech Ad ID service - meaning they have to contact us, enter into a contract with us and go through some technical integration. As a result, we know exactly who uses this ID. Therefore this renaming will also be coordinated with everyone involved. Does the major version rule still apply, or can this be excluded, as a special case? Thanks.

**Comment by bretg** [comment]
Will let @patmmccann and the committee decide if we can make an exception

**Comment by EskelCz** [comment]
@patmmccann So, any verdict on this? :) Thanks

**Comment by patmmccann** [comment]
Can you provide a list of existing users?

**Comment by EskelCz** [comment]
@patmmccann According to the project coordinator, at the moment the only partner using the module is Czech News Center a.s. We're in close contact with them and the system is not yet in production.

**Comment by patmmccann** [comment]
okay

**Review by patmmccann — APPROVED** [review]

**Pull request merged**